include/simdjson/generic/numberparsing.h

```diff
@@ -509,16 +509,17 @@ simdjson_inline error_code write_float(const uint8_t *const src, bool negative,
     // we are almost never going to get here.
     if (!parse_float_fallback(src, &d)) { return INVALID_NUMBER(src); }
   }
-  WRITE_DOUBLE(d, src, writer);
+  WRITE_DOUBLE(d, src, writer, index);
   return SUCCESS;
 }
 
 // for performance analysis, it is sometimes  useful to skip parsing
 #ifdef SIMDJSON_SKIPNUMBERPARSING
+  // TODO: update these
```

Comment:
I'm unfamiliar with what `these` is referring to - is this TODO complete or incomplete?